packages/core/src/llm/model/gateways/models-dev.ts

```diff
@@ -95,23 +96,25 @@ export class ModelsDevGateway extends MastraModelGateway {
         providerInfo.npm === '@ai-sdk/gateway' || // Vercel AI Gateway is OpenAI-compatible
         normalizedId in OPENAI_COMPATIBLE_OVERRIDES;
 
+      const isOtherProvider = SUPPORTED_PROVIDERS.includes(providerId);
```

Comment:
hmm it's not clear to me what `isOtherProvider` mean, what is the "other" in this case?